src/instructions/auf_dem_trockenen.md

```diff
@@ -70,11 +66,11 @@ The _Histogram_ operator can be used to visualise the distribution of occurrence
 
 <img src="images/auf_dem_trockenen_14.png" width="50%" height="auto" alt="Histogram">
 
-To visualise the classified data, it is recommended to use the _Class Histogram_ operator, which translates the Ecosystem Atlas numbers into class names using the metadata.
+To visualise the classified data, it is recommended to use the _Class Histogram_ operator, which translates the IÖR land classification numbers into class names using the metadata.
```

Comment:
Not important, but you could replace the "visualise" with "visualize" at multiple points in the documentation since the VAT title uses this spelling to make it uniform